city/models.py

```diff
@@ -0,0 +1,14 @@
+from sqlalchemy import Column, Integer, String
```

Comment:
Ensure that the `Base` class is imported correctly from your database module. This is crucial for the SQLAlchemy ORM to function properly.